INSTALL.rst

```diff
@@ -76,3 +76,21 @@ To use debug information in PySCIPOpt you need to build it like this:
 
 Be aware that you will need the **debug library** of the SCIP Optimization
 Suite for this to work (``cmake .. -DCMAKE_BUILD_TYPE=Debug``).
+
+Testing new installation
+========================
+
+To test your brand-new installation of PySCIPOpt you need `pytest <https://pytest.org/>`__ on your system:
+
+::
+
+    sudo apt-get install python-pytest   # for Python 2, on Linux
+    sudo apt-get install python3-pytest  # for Python 3, on Linux
+
+Tests are in the ``tests`` directory. Any test can be run with:
+::
+
+   python -m pytest test_name.py
```

Comment:
It's easier to run `py.test` or `py.test tests/test_*.py` to run only a specific test.